atkinson/errors/reporters/trello.py

```diff
@@ -239,3 +239,4 @@ def close(self):
         if current_column != self.__close_column:
             self.__api.cards.update_idList(self.__card_id,
                                            self.__close_column)
+            self.__api.cards.update_pos(self.__card_id, 'bottom')
```

Comment:
is this bit something that can/should be tested?

Reply:
hmm, I think I can at least add a check to ensure the trello api is getting called.